### DISCUSSION TITLE
supabase IS NOT LOADING on Mac, safari 13.1.1 Mac OS Catalina

### DISCUSSION THREAD

**Discussion by bret1234** [supabase/supabase#6078] [Questions]
im trying to log into the app.supabase.io, it goes to the website, BUT ITS BLANK, 
its on the site, I can tell that the background is there, but nothing else loads. 

what causes that?

ive cleared my cache and data from the site, and it still doesn't load

**Comment by GaryAustin1** [comment]
Possibly related issue?
https://github.com/supabase/supabase/issues/6079

**Reply by bret1234** [reply]
> Possibly related issue?
> 
> https://github.com/supabase/supabase/issues/6079

What issue is it?

**Reply by GaryAustin1** [reply]
A new issue reported with Mac and blank screen at the link...

"On one Mac, it loads fine, but my 2 other Macs, running Mojave, the website dosnt load, it's a grey blank screen that dosnt load anything. I open the web inspector, and it's a forever increasing ERROR"

This is the 3rd Mac issue I've seen today with Supabase loading, although one is on discord and could be one of you two posting there...
